src/main/webapp/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
This isn't quite right. By placing the MIT license here, you would be applying this MIT license (with copyright "Start Bootstrap LLC") to all files in this directory, which is supposed to be your code.

Instead, you should create a new directory (we can call it `src/main/webapp/vendor`), where you should place both this LICENSE file, and also the index.html file (since that was derived from their code). Future updates to index.html would happen in `vendor/`. For any other HTML templates you create on your own, you can place those in `src/main/webapp/` proper.

You can (hopefully) see more guidance here: https://groups.google.com/a/google.com/g/third-party-help/c/uVob1Bnqwv0/m/Gj8TdWVbCQAJ. If you can't, the short version is noted in the first paragraph of my comment here.